### PR TITLE
Fix the copyright heading linter

### PR DESCRIPTION
**Pull request by milanmlft**
Run only on git-staged files

Fixing my overenthusiastic merge of  #317

### CODE REVIEW

**Comment by peshence** [comment]
I know it happened in the previous commit, but I would have the CI on just in case, i doubt it's an expensive job.

Also you've committed some readme about configs?

**Comment by jeremyestein** [comment]
> I know it happened in the previous commit, but I would have the CI on just in case, i doubt it's an expensive job.

It's running from the pre-commit command, so it's still part of the "lint" build job.

File: README.md [hunk 1]
```diff
@@ -188,6 +188,38 @@ For the deidentification of the EHR extracts, we rely on an instance running the
 [CogStack API](https://cogstack.org/) with a `/redact` endpoint. The URL of this instance should be
 set in `.env` as `COGSTACK_REDACT_URL`.
 
+### 3. Configure a new project
```
Comment (stefpiatek) [inline README.md:191]: did you want this to be in here?

Comment (milanmlft) [reply README.md:191]: Oh ffs that's where that went 🤦‍♂️ 🤦‍♂️ 🤦‍♂️ 🤦‍♂️  I thought I'd lost that
> Note to self: `git stash --all` before switching branches

File: README.md [hunk 2]
```diff
@@ -142,7 +142,7 @@ setfacl -R -m d:g::rwX /gae/pixl_dev
 
 ### 1. Choose deployment environment
 
-This is one of dev|test|staging|prod and referred to as `<environment>` in the docs.
+This is one of `dev|test|staging|prod` and referred to as `<environment>` in the docs.
```
Comment (jeremyestein) [inline README.md:145]: This change also looks unrelated

**Review by stefpiatek — APPROVED** [review]
